assets/blocks/reader-registration/index.php

```diff
@@ -180,18 +180,38 @@ function render_block( $attrs ) {
 						</div>
 					<?php endif; ?>
 					<div class="newspack-registration__main">
-						<div>
-							<div class="newspack-registration__inputs">
-								<input type="email" name="email" autocomplete="email" placeholder="<?php echo \esc_attr( $attrs['placeholder'] ); ?>" />
-								<input type="submit" value="<?php echo \esc_attr( $attrs['label'] ); ?>" />
-							</div>
-							<div class="newspack-registration__privacy">
-								<p>
-									<?php echo $attrs['privacyLabel']; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
-								</p>
+						<div class="newspack-registration__inputs">
+							<input type="email" name="email" autocomplete="email" placeholder="<?php echo \esc_attr( $attrs['placeholder'] ); ?>" />
+							<input type="submit" value="<?php echo \esc_attr( $attrs['label'] ); ?>" />
+						</div>
+
+						<?php if ( Newspack\Google_OAuth::is_oauth_configured() ) : ?>
+							<div class="newspack-registration__logins">
+								<div class="newspack-registration__logins__separator">
+									<div></div>
+									<div>
+										<?php echo \esc_html__( 'OR', 'newspack' ); ?>
+									</div>
+									<div></div>
+								</div>
+								<button class="newspack-registration__logins__google">
+									<?php echo file_get_contents( dirname( NEWSPACK_PLUGIN_FILE ) . '/assets/blocks/reader-registration/icons/google.svg' ); // phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped ?>
+									<span>
+										<?php echo \esc_html__( 'Sign in with Google', 'newspack' ); ?>
+									</span>
+								</button>
 							</div>
+						<?php endif; ?>
```

Comment:
This section should be inside `newspack-registration__inputs` to not be vertically centered on the flex layout of `newspack-registration__main`.

<img width="1252" alt="image" src="https://user-images.githubusercontent.com/820752/180273811-9142b0fb-12bb-480f-aa50-50b472b50db8.png">